### DISCUSSION TITLE
👋 Welcome to MIDeL!

### DISCUSSION THREAD

**Discussion by BardiaKh** [Mayo-Radiology-Informatics-Lab/MIDeL#9] [General]
We’re using Discussions as a place to connect with other members of the MIDeL community. We encourage you to:

- Ask questions you’re wondering about.
- Share chapter ideas.
- Engage with other community members.

To get started, comment below with an introduction of yourself and tell us about what will you be using MIDeL for.

**Comment by BardiaKh** [comment]
Starting the thread:

I'm Bardia, a medical doctor working at @Mayo-Radiology-Informatics-Lab. My work focuses on applying deep learning to musculoskeletal imaging, specifically, post-op risk prediction. I am also interested in image generation tasks to augment model performance.

Happy to meet you all. 🙌

**Comment by madhavajay** [comment]
Hi, this looks great. ❤️

Is there anywhere to subscribe to get updates as it looks like the first pass of content is not quite finished?
The twitter link doesn't go anywhere either, is there a best place to follow for updates?

**Comment by slowvak** [comment]
You are right that this is not ‘finished’ but I think it probably never will be—the science will keep advancing and we hope to keep addressing these new things. We will send out tweets on our handle as new content is added. Please provide feedback if you see things that are confusing (and also things you think are well done so we know to keep those).
BJE

**Reply by madhavajay** [reply]
Hi @slowvak do you know which twitter account? The link doesn't work on the MIDeL page.

**Reply by BardiaKh** [reply]
Thanks for reporting this issue. We have fixed the link. Our twitter handler is [@MayoAILab](https://twitter.com/MayoAILab).
